VisualPIC/DataHandling/unitConverters.py

```diff
@@ -317,8 +318,14 @@ def SetSimulationParameters(self, params):
         super().SetSimulationParameters(params)
         self._SetNormalizationFactor(None)
 
-    def ConvertToISUnits(self, dataElementName, data):
-        return data
+    def ConvertToISUnits(self, dataElement, data):
+        dataElementName = dataElement.GetName()
+        if dataElementName == "x" or dataElementName == "y" or dataElementName == "z":
```

Comment:
better rely on the `unitDimensions` attribute of the parent folder of `dataElement` (for non-scalar records)